src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java

```diff
@@ -27,7 +27,7 @@
 import java.io.IOException;
 import java.security.NoSuchAlgorithmException;
 
-import static org.fundacion.jala.converter.service.ChecksumService.getFileChecksum;
+import static org.fundacion.jala.converter.core.ChecksumService.getFileChecksum;
```

Comment:
in that file the lines 18 and 29 should be removed

Reply:
Solved.